src/poggit/release/submit/SubmitModule.php

```diff
@@ -0,0 +1,894 @@
+plugin.
+EOD
+            ,
+            "refDefault" => $this->refRelease->shortDesc,
+            "srcDefault" => $this->pluginYml["description"] ?? ($this->repoInfo->description ?: null)
+        ];
+        $fields["version"] = [
+            "remarks" => <<<EOD
+The version of this release. The version <em>must be named according to <a target="_blank" href="http://semver.org">Semantic Versioning</a></em>,
+i.e. the version must consist of two or three numbers, optionally with prerelease information behind a hyphen, e.g.
+<code>1.0</code>, <code>2.0.1</code>, <code>3.0.0-beta</code>, <code>4.7.0-beta.3</code>. Note that adding build
+metadata behind a <code>+</code> in the version is discouraged due to URL encoding inconvenience.<br/>
+This version will replace the <code>version</code> attribute in plugin.yml in the release phar, so this doesn't have to
+be same as that in plugin.yml.
+EOD
+            ,
+            "refDefault" => $this->refRelease->version,
+            "srcDefault" => $this->pluginYml["version"] ?? null
+        ];
+        $fields["description"] = [
+            "remarks" => <<<EOD
+A detailed description of your release. You may link to other sites for documentation in this page, but you must include
+a basic description here in case the other sites are down.<br/>
+It is recommended that the description includes the following:<br/>
+<ul>
+    <li>Features (what this plugin does)</li>
+    <li>Usage (command syntax, or other special ways you need to use to interact with this plugin)</li>
+    <li>Protips (if the usage of this plugin is not straightforward, you can give a few ideas how this plugin can be used)</li>
+</ul>
+While you may import README from your repo, make sure you don't accidentally include irrelevant things like download
+links in the description.<br/>
+Plugins with insufficient or irrelevant description may be rejected.
+EOD
+            ,
+            "refDefault" => $this->refRelease instanceof stdClass ? [
+                "type" => $this->refRelease->desctype === "html" ? "sm" : $this->refRelease->desctype,
+                "text" => $this->refRelease->desctype === "html" && $this->refRelease->descrMd !== null ?
+                    ResourceManager::read($this->refRelease->descrMd, "md") : ResourceManager::read($this->refRelease->description, $this->refRelease->desctype)
+            ] : null,
+            "srcDefault" => null
+        ];
+        $fields["license"] = [
+            "remarks" => <<<EOD
+The license your plugin is released with.<br/>
+You should use the same one used in your source code.
+EOD
+            ,
+            "refDefault" => $this->refRelease instanceof stdClass ? [
+                "type" => $this->refRelease->license,
+                "custom" => $this->refRelease->licenseRes === null ? null : ResourceManager::read($this->refRelease->licenseRes, "md")
+            ] : null,
+            "srcDefault" => [
+                "type" => $this->repoInfo->license->key,
```

Comment:
I  get 'Trying to get property of non-object' here when loading an existing release